.github/CODEOWNERS

```diff
@@ -45,6 +45,7 @@
 /federatorai/                            @chenyingtz @yummydsky
 /filebeat/                               @wk8 jean@tripping.com
 /finout/                                 @roiravhon support@finout.io
+/instabug/                               @AbdallahHemdan ahemdan@instabug.com
```

Comment:
Could you also add instabug in [this section](https://github.com/DataDog/integrations-extras/blob/master/.github/CODEOWNERS#L174) so our doc team gets notified when the README, metadata or manifest files are modified?